src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -54,6 +59,41 @@ public static string LocalSlotName
 
 #pragma warning disable SA1117 // Parameters should be on same line or separate lines
 
+        private void SanitizeString(string? rawPayload, out string iloggerString, out string durableKustoTableString, bool isReplay = false)
```

Comment:
Do we have unit or E2E tests to check if we're logging the expected information correctly?

Reply:
Yes, added some unit tests for the sanitizers here: https://github.com/Azure/azure-functions-durable-extension/pull/2844/commits/e16b9126e6e20bb2ec24e96b65ce7fc5fdaeb17d